src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.MonoCrossAOT.sfxproj

```diff
@@ -30,13 +31,7 @@
       <_SdkPropsProperties Include="TargetRid" Value="$(TargetCrossRid)" />
     </ItemGroup>
     <PropertyGroup>
-      <_PermissionsFiles>&lt;File Path=&quot;tools/mono-aot-cross&quot; Permission=&quot;755&quot; /&gt;</_PermissionsFiles>
-    </PropertyGroup>
-    <PropertyGroup Condition="(!$([MSBuild]::IsOsPlatform('Windows')) and '$(TargetCrossRid)' != 'browser-wasm')">
-      <_PermissionsFiles>$(_PermissionsFiles)
-    &lt;File Path=&quot;tools/opt&quot; Permission=&quot;755&quot; /&gt;</_PermissionsFiles>
-      <_PermissionsFiles>$(_PermissionsFiles)
-    &lt;File Path=&quot;tools/llc&quot; Permission=&quot;755&quot; /&gt;</_PermissionsFiles>
+      <_PermissionsFiles>@(_ToolFile -> '&lt;File Path=&quot;tools/%(RecursiveDir)%(FileName)%(Extension)&quot; Permission=&quot;755&quot; /&gt;', ' ')</_PermissionsFiles>
```

Comment:
I think there might be some json files in there, may need to condition it on %(Extension) == ' '

Reply:
It looks like a step in the installer is adding the json files, so we should be good.